docs/docs/examples/property_graph/property_graph_advanced.ipynb

```diff
@@ -0,0 +1,345 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    )
</pre><p>out of curiosity can this extractor be used on its own? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
it sure can -- `nodes = kg_extractor(nodes)`  -- since its a transform component, you can just call it. Its inserting the nodes and relations into the metadata